docs/kqueen.rst

```diff
@@ -442,6 +442,41 @@ Provision a Kubernetes cluster using Google Kubernetes Engine
 #. Click ``Submit``.
 #. To track the cluster status, navigate to the KQueen main dashboard.
 
+Provision a Kubernetes cluster using Openstack Kubespray Engine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Log in to Openstack Horizon Dashboard.
+#. Select your Project.
+#. Navigate to Project`-> ``Compute`` -> ``Access & Security`` -> ``API Access`` tab and click
+   ``Download OpenStack RC File``.
+#. Log in to the KQueen web UI.
+#. From the ``Create Provisioner`` tab, select ``Openstack Kubespray Engine``.
+#. Specify provisioner requirements with the Openstack RC File, downloaded earlier.
+#. Click ``Deploy Cluster``.
+#. Select the defined Openstack provisioner.
+#. Specify the cluster requirements, and pay attention on following cases:
+
+   #. ``Ssh key name`` is name of SSH key pair. You should choose from existing pairs or create a new one.
+   #. ``Image name`` should be compatible with `Kubespray requirements <https://github.com/kubernetes-incubator/kubespray>`_.
+   #. ``Flavor`` should be at least ``m1.small`` (minimum 2GB RAM).
+   #. ``Ssh username`` is login username for nodes. Depends on defined ``Image name`` 
+
+#. Click ``Submit``.
+#. To track the cluster status, navigate to the KQueen main dashboard.
```

Comment:
I think a user will be redirected to the main dashboard automatically. Maybe it's better to note that the cluster info can be found on a cluster page or smth.